actors/builtin/miner/miner_test.go

```diff
@@ -1207,7 +1207,7 @@ func TestWindowPost(t *testing.T) {
 		actor.submitWindowPoSt(rt, dlinfo, partitions, infos, cfg)
 
 		// expect declared fee to be charged during cron
-		dlinfo = advanceDeadline(rt, actor, &cronConfig{ongoingFaultsPenalty: declaredFee})
+		dlinfo = advanceDeadline(rt, actor, &cronConfig{declaredFaultsPenalty: declaredFee})
```

Comment:
I see some of my earlier confusion now. Here, the original is better IMO. Nothing is being declared, and some of the power may have been an undeclared fault.